butterfree/extract/readers/reader.py

```diff
@@ -106,6 +108,9 @@ def build(
             transformed_df = self.incremental_strategy.filter_with_incremental_strategy(
                 transformed_df, start_date, end_date
             )
+
+        self.run_post_hooks(transformed_df)
```

Comment:
As I mentioned earlier you need to assign a variable to get the return of the method `run_post_hooks` and `run_pre_hooks`, since they don't update the dataframe in place. 😬 You'll need to change that in all uses in the other files too.